OpenKh.Tools.Kh2TextEditor/ViewModels/TextEditorViewModel.cs

```diff
@@ -127,6 +139,66 @@ public TextEditorViewModel()
             Drawing = new SpriteDrawingDirect3D();
             CurrentMessageEncoder = Encoders.InternationalSystem;
             _messages = new MessagesModel(this, this, new Msg.Entry[] { });
+
+            HandleReturn = new RelayCommand(x =>
+            {
+                Text += "{:newline}";                
+                SelectedItem = SelectedItem;
+            });
+
+            HandleRemoval = new RelayCommand(x =>
+            {
+                MessageEntries.RemoveAll(x => x.Id == SelectedItem.Id);
+                ResetMessagesView();
+            });
+
+            HandleAddition = new RelayCommand(x =>
+            {
+                int _id = MessageEntries[MessageEntries.Count - 1].Id;
+                MessageEntries.Add(new Msg.Entry() { Id = _id + 1, Data = new byte[] { 0x33, 0x2E, 0x38, 0x32, 0x00 } });
+                ResetMessagesView();
+            });
+
+            HandleComm = new RelayCommand(x =>
+            {
+                int _o = Convert.ToInt32(x);
+
+                switch (_o)
+                {
+                    case 0:
+                        Text += "{:reset}";
+                        break;
+                    case 1:
+                    {
+                        Color _color;
+                        bool _ok = ColorPickerWindow.ShowDialog(out _color);
+
+                        if (_ok)
+                            Text += "{:color " + string.Format("#{0}{1}{2}{3}", _color.R.ToString("X2"), _color.G.ToString("X2"), _color.B.ToString("X2"), _color.A.ToString("X2")) + "}";
+                    }
+                        break;
+                    case 2:
+                        Text += "{:scale 16}";
+                        break;
+                    case 3:
+                        Text += "{:width 100}";
+                        break;
+                    case 4:
+                        Text += "{:clear}";
+                        break;
+                    case 5:
+                        Text += "{:position 0,0}";
+                        break;
+                }
+
+                SelectedItem = SelectedItem;
```

Comment:
Same here for the suggestion I written above

Reply:
Same as I mentioned above.